docs/FAQ/gui-faq-en.md

```diff
@@ -20,147 +20,155 @@ Zonemaster
 
 <a name="q1"></a>
 #### 1. What is Zonemaster?
-Zonemaster is a program that was designed to help people check, measure and
-hopefully also understand how DNS (Domain Name System) works.
+Zonemaster is a program designed to help people check, measure and
+hopefully also understand how the DNS (Domain Name System) works.
 
-It consists of four basic parts:
+It consists of four components:
 
   1. Engine - a test framework that supports all functionality to perform DNS tests.
-  2. The CLI interface to the Engine.
-  3. Backend, a server that allows you to run Zonemaster tests and save results using
+  2. CLI - a command-line interface to the Engine.
+  3. Backend - a server that allows you to run Zonemaster tests and save results using
      a JSON-RPC API and a database.
-  4. GUI - a web interface to the backend.
+  4. GUI - a web interface to the Backend.
 
-When a domain (such as "zonemaster.net") is submitted to Zonemaster (CLI or
-GUI) it will investigate the domain’s general health by traversing the DNS from root
-(.) via the TLD (Top Level Domain, like .net) to the nameserver(s) that hosts
-the information about the specific domain (zonemaster.net). The different sanity checks
-conducted by the Zonemaster tool is documented in the [Test Requirements
-document](https://github.com/zonemaster/zonemaster/blob/master/docs/requirements/TestRequirements.md).
+When a domain (such as 'zonemaster.net') is submitted to Zonemaster (CLI or
+GUI) it will investigate the domain’s general health with a series of tests.
```

Comment:
Maybe “ascertain” instead of “investigate”?

Reply:
"verify"?

In general, "domain name" is probably better than "domain".

Reply:
I went with "verify" to keep it simple. Updated.